modules/nlu/src/views/full/common/TrainingControl.tsx

```diff
@@ -0,0 +1,18 @@
+import { NLUApi } from 'api'
+import React, { FC } from 'react'
+
+import style from './style.scss'
+import AutotrainToggle from './AutotrainToggle'
+import TrainNow from './TrainNow'
+
+const TrainingControl: FC<{ api: NLUApi }> = ({ api }) => {
+  return (
+    <div className={style.trainingControl}>
+      <h5>Training</h5>
+      <AutotrainToggle api={api} />
+      <TrainNow api={api} />
+    </div>
```

Comment:
I find the h5 "training" tag a big redundant. Also, can you change the intent of the button when it is toggled? Intent.PRIMARY when active, Intent.NONE otherwise. Since the icon is the opposite of what's actually happening, it could give a clue